plugin/src/_variants/spaceAndDivide.js

```diff
@@ -0,0 +1,10 @@
+export const variantSpaceAndDivide = (matcher) => {
+  if (/^space-?([xy])-?(-?.+)$/.test(matcher) || matcher.startsWith('divide-')) {
```

Comment:
Another question here is why can `([xy])` and the next group (`(-?.+)` be optional when the rules implemented in `_rules/space-margin` seem to always expect them? I mean that we do not support `space-2` or just `space` but always `space-x-2`, `space-y-144` or `space-y-reverse`